generators/dart/math.js

```diff
@@ -204,35 +244,25 @@ Blockly.Dart['math_number_property'] = function(block) {
          '  }',
          '  return true;',
          '}']);
-    code = functionName + '(' + number_to_check + ')';
-    return [code, Blockly.Dart.ORDER_UNARY_POSTFIX];
-  }
-  switch (dropdown_property) {
-    case 'EVEN':
-      code = number_to_check + ' % 2 == 0';
-      break;
-    case 'ODD':
-      code = number_to_check + ' % 2 == 1';
-      break;
-    case 'WHOLE':
-      code = number_to_check + ' % 1 == 0';
-      break;
-    case 'POSITIVE':
-      code = number_to_check + ' > 0';
-      break;
-    case 'NEGATIVE':
-      code = number_to_check + ' < 0';
-      break;
-    case 'DIVISIBLE_BY':
-      const divisor = Blockly.Dart.valueToCode(block, 'DIVISOR',
-          Blockly.Dart.ORDER_MULTIPLICATIVE);
-      if (!divisor) {
-        return ['false', Blockly.Dart.ORDER_ATOMIC];
-      }
```

Comment:
Missing from updated code.

Reply:
But also note that by adding `|| '0'` to the original line [228](https://github.com/RichDoherty/blockly/pull/9/commits/f64d508970df4371fb7c290da52133e4249cf04b#diff-7583a85dd490bcc2832d4b6b182f919f0a524a4b1eaf11aa518246809c675759L228), you make it so that `divisor` is always a truthy value whereby `!divisor` will always be `false` (the string `'0'` is a truthy value).